web/public/mgmt/config.yml

```diff
@@ -999,11 +999,6 @@ collections:
                   - { label: "Optional field label", name: "optionalLabel", widget: "string" }
                   - { label: "Generic inline error text", name: "genericErrorText", widget: "string" }
                   - { label: "Maximum length error text", name: "maximumLengthErrorText", widget: "string" }
-                  - {
-                      label: "Partially entered address error text",
-                      name: "partialAddressErrorText",
-                      widget: "string",
```

Comment:
only thing i touched, i guess the other stuff is content changes?

Reply:
It looks like whitespace changes. I pulled the branch and just saved the file and my editor reverted. Not a huge deal, but maybe try running the formatter on the file to reduce the diff.

Reply:
you sir, are the man!